File: Back/ApiTcg/Domain/Entities/Card.cs
namespace Domain.Entities;

public class Card
{
    public string Id { get; set; } = "";
    public string Name { get; set; } = "";
    public string SetId { get; set; } = "";       
    public string? LocalId { get; set; }          
    public string? Image { get; set; }


    public Set Set { get; set; } = new Set();
    public ICollection<Collection> Collections { get; set; } = [];
}

File: Front/poke-tcg/src/app/models/card/card.ts
import { CardPriceSummary } from "../cardPrice/card-price-summary";
import { Collection } from "../collection/collection";
import { SetSummary } from "../set/set-summary";

export interface Card
{
  id: string;
  name: string;
  setId: string;
  localId: string;
  image?: string;
  set: SetSummary;
  collection?: Collection;
  price?: CardPriceSummary;
}
